libs/MeshKernel/tests/src/Mesh2DTest.cpp

```diff
@@ -77,47 +77,6 @@ TEST(Mesh2D, OneQuadTestConstructor)
     ASSERT_EQ(meshkernel::constants::missing::sizetValue, mesh.m_edgesFaces[3][1]);
 }
 
-TEST(Mesh2D, TriangulateSamplesWithSkinnyTriangle)
-{
-    // Prepare
-    std::vector<meshkernel::Point> nodes;
-
-    nodes.push_back({302.002502, 472.130371});
-    nodes.push_back({144.501526, 253.128174});
-    nodes.push_back({368.752930, 112.876755});
-    nodes.push_back({707.755005, 358.879242});
-    nodes.push_back({301.252502, 471.380371});
-    nodes.push_back({302.002502, 472.130371});
-
-    meshkernel::Polygons polygons(nodes, meshkernel::Projection::cartesian);
-
-    // Execute
-    const auto generatedPoints = polygons.ComputePointsInPolygons();
-
-    meshkernel::Mesh2D mesh(generatedPoints[0], polygons, meshkernel::Projection::cartesian);
-
-    // Assert
-    ASSERT_EQ(6, mesh.GetNumEdges());
-
-    ASSERT_EQ(4, mesh.m_edges[0].first);
-    ASSERT_EQ(1, mesh.m_edges[0].second);
-
-    ASSERT_EQ(1, mesh.m_edges[1].first);
-    ASSERT_EQ(2, mesh.m_edges[1].second);
-
-    ASSERT_EQ(2, mesh.m_edges[2].first);
-    ASSERT_EQ(4, mesh.m_edges[2].second);
-
-    ASSERT_EQ(0, mesh.m_edges[3].first);
-    ASSERT_EQ(2, mesh.m_edges[3].second);
-
-    ASSERT_EQ(2, mesh.m_edges[4].first);
-    ASSERT_EQ(3, mesh.m_edges[4].second);
-
-    ASSERT_EQ(3, mesh.m_edges[5].first);
-    ASSERT_EQ(0, mesh.m_edges[5].second);
-}
-
 TEST(Mesh2D, TriangulateSamples)
```

Comment:
This unit test does not seem to test anything.

Reply:
it is checking the correct triangulation is produced